src/india_api/internal/service/auth.py

```diff
@@ -0,0 +1,60 @@
+""" Authentication  objects """
```

Comment:
Switch to use https://github.com/openclimatefix/pv-site-api/blob/main/pv_site_api/auth.py